lib/interceptor/dump.js

```diff
@@ -93,58 +91,33 @@ class DumpHandler extends DecoratorHandler {
 
     if (this.#size >= this.#maxSize) {
       this.#dumped = true
-
-      if (
-        this.#abortOnDumped &&
-        (!this.#waitForTrailers || !this.#hasTrailers)
-      ) {
-        this.#reason = new RequestAbortedError(
-          `Response dumped (${this.#size}) for max size (${this.#maxSize})`
-        )
-
-        this.#abort(this.#reason)
-        return false
-      }
+      this.#handler.onData('')
```

Comment:
This is strange?

Reply:
Otherwise the request keeps hanging, tried a workaround without luck; find it quite counterintuitive but most likely is something that might need to be addressed at the `request` level, but not sure the scope of this PR

Reply:
I'd like to address this before landing. Do you have minimal a repro? I can take a look.

Reply:
nvm, I found the issue; it was on me, the request was in the aborted state and couldn't continue because already aborted but I never called the `onError` hook from the original handler. That made the work